src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  /* font-size: 0; */
+}
+
+/* comment .stars and uncomment the others, font-size metters there which i am curios about, why?*/
+.stars {
+  display: flex;
+  flex-flow: row wrap;
+  width: 100px;
+}
+
+.stars__star {
+  /* display: inline-block; */
+  background-image: url(images/star.svg);
+  background-size: cover;
+  height: 12px;
+  width: 12px;
+  margin: 2px 6px 2px 2px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
what will happen if I decide to change the number of stars in one row to, let's say, 7?